### PR TITLE
fix emoji encode error

### LINKED ISSUE #129: fix:go emoji to java emoji（a character takes up 3-4 bytes of special symbols）
**What would you like to be added**:

**Why is this needed**:
image

### PR DESCRIPTION
**Pull request by wongoo**
**What this PR does**:
Fix emoji encode error

**Which issue(s) this PR fixes**:
Fixes  #129 

**Special notes for your reviewer**:
format code using:
```
goimports -w -l .
go fmt ./...
```

```release-note
- Fix emoji encode error
```

### CODE REVIEW

**Comment by wongoo** [comment]
@fangyincheng @pantianying @aliiohs @AlexStocks  pls review

File: test_hessian/src/main/java/test/TestString.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test;
+
+/**
+ * @author wongoo
```
Comment (lovepoem) [inline test_hessian/src/main/java/test/TestString.java]: Please remove author

Comment (wongoo) [reply test_hessian/src/main/java/test/TestString.java]: done

File: test_hessian/src/main/java/test/HessianTool.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import com.caucho.hessian.io.SerializerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+/**
+ * @author wongoo
+ */
+public class HessianTool {
```
Comment (fangyincheng) [inline test_hessian/src/main/java/test/HessianTool.java]: are `HessianTool` and `HexTool`  already used？

Comment (wongoo) [reply test_hessian/src/main/java/test/HessianTool.java]: not used for unit test, but often used when testing some hessian data

**Comment by wongoo** [comment]
> I test this code with some of my practical cases，and the results were in line with expectations.

@pantianying can u provide a unit test

**Comment by wongoo** [comment]
> I test this code with some of my practical cases，and the results were in line with expectations.

@pantianying  thanks for testing

**Pull request merged**